### PR TITLE
chore(PreviewTools): Set controls theme to light

### PR DESCRIPTION
**Pull request by pladaria**
Controls theme was not forced to light, this fixes it.
Verified on Mac safari, using dark/light modes

### CODE REVIEW

**Review by yceballost — APPROVED** [review]
working perfectly in Brand Factory

File: playroom/components.tsx [hunk 1]
```diff
@@ -89,7 +88,7 @@ const useStyles = createUseStyles(() => ({
     },
 }));
 
-const themesMap: {[skinName in SkinName]: {themeConfig: ThemeConfig; text: string}} = {
+const themesMap: {[skinName: string]: {themeConfig: ThemeConfig; text: string}} = {
```
Comment (pladaria) [inline playroom/components.tsx:91]: TS was reporting this as an error, because O2-classic is missing in the themesMap object

Comment (atabel) [inline playroom/components.tsx:91]: why this change?

**Review by atabel — APPROVED** [review]

File: playroom/components.tsx (hunk 1 above)
Comment (atabel) [reply playroom/components.tsx:91]: oh, there is a way to make this non-exhaustive?

Comment (atabel) [reply playroom/components.tsx:91]: also, why don't we run tsc on CI? :/

Comment (pladaria) [reply playroom/components.tsx:91]: we are running tsc but it only checks files used by the library
